lib/dal/src/workspace_snapshot/graph.rs

```diff
@@ -996,7 +996,8 @@ impl WorkspaceSnapshotGraph {
         debug!("only to rebase edges: {:?}", &only_to_rebase_edges);
         debug!("only onto edges: {:?}", &only_onto_edges);
 
-        let root_seen_as_of_onto = self
+        // This is the last time that to_rebase was written to by the onto change set
```

Comment:
Shouldn't this say `... the last time that to_rebase knows about onto seeing the state of the graph`? `onto` won't necessarily have made any changes other than recording that it saw things.